src/seedu/addressbook/commands/TopFiveCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Lists the first five persons in the address book to the user.
+ */
+
+public class TopFiveCommand extends Command{
+    public static final String COMMAND_WORD = "topFive";
```

Comment:
maybe to keep the command word format consistent, you can use top_five instead of camelCase, or simply top might be more intuitive and user-friendly

Reply:
alternatively, you might want to allow the user to specify how many persons to list, so command like 'top 5'